src/tests/eina/eina_test_file.c

```diff
@@ -491,7 +491,7 @@ static void *
 _eina_test_file_thread(void *data EINA_UNUSED, Eina_Thread t EINA_UNUSED)
 {
 #ifdef _WIN32
-   const char *filename = "cmd.exe";
+   const char *filename = "C:/Windows/system.ini";
```

Comment:
What is this file?

Reply:
Just a test file with an absolute path for test